src/init.cpp

```diff
@@ -333,7 +333,7 @@ void SetupServerArgs()
     gArgs.AddArg("-version", "Print version and exit", false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-alertnotify=<cmd>", "Execute command when a relevant alert is received or we see a really long fork (%s in cmd is replaced by message)", false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-assumevalid=<hex>", strprintf("If this block is in the chain assume that it and its ancestors are valid and potentially skip their script verification (0 to verify all, default: %s, testnet: %s)", defaultChainParams->GetConsensus().defaultAssumeValid.GetHex(), testnetChainParams->GetConsensus().defaultAssumeValid.GetHex()), false, OptionsCategory::OPTIONS);
-    gArgs.AddArg("-blocksdir=<dir>", "Specify blocks directory (default: <datadir>/blocks)", false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-blocksdir=<dir>", "Specify blocks root directory for .dat files (default: <datadir>)", false, OptionsCategory::OPTIONS);
```

Comment:
Could reference block data files as described in [doc/files.md](https://github.com/bitcoin/bitcoin/blob/master/doc/files.md): `blocks/*.dat` ?

Reply:
Good idea but I'm not sure the best way to phrase that... `Specify root directory for blocks/*.dat files...`?

Reply:
I would reed this as the .dat files will be written to the "root directory" (but they get written into "<root>/blocks"). I think mentioning the "blocks" subdirectory (that will e created) makes sense.

Reply:
```
"Specify directory to hold blocks subdirectory for *.dat files (default: <datadir>)"
```